packages/mc-scripts/src/commands/start-vite.ts

```diff
@@ -111,6 +112,26 @@ async function run() {
           plugins: ['@emotion/babel-plugin'],
         },
       }),
+      pluginSvgr({
```

Comment:
I was wondering if we also need to include this plugin in the `build-vite` command as well.

Reply:
Good point, yes we do. I forgot.